src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -138,6 +139,148 @@ void RotatedSPOs::resetParametersExclusive(const opt_variables_type& active)
   }
 }
 
+void RotatedSPOs::writeVariationalParameters(hdf_archive& hout)
+{
+  hout.push("RotatedSPOs");
+  if (use_global_rot_)
+  {
+    hid_t grp                   = hout.push("rotation_global");
+    std::string rot_global_name = std::string("rotation_global_") + SPOSet::getName();
+
+    int nparam = myVarsFull.size();
```

Comment:
Could you add const if the variable won't be modified?

Reply:
I used `nparam` as a shortcut name for `myVarsFull.size()` here and as a shortcut name for `myVars.size()` later.  While these are technically in different scopes and the compiler is okay with the name reuse, it might be confusing to the reader.  I update the shortcut name to `nparam_full` when referring to `myVarsFull.size()`.

Reply:
That is exactly the purpose of using const to enhance readability. I have seen temp being reused in a similar way and completely destroys the readability.